account_invoice_import_invoice2data/security/res_groups.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="base.group_user" model="res.groups">
+            <field name="implied_ids" eval="[(4, ref('base.group_user'))]" />
```

Comment:
Wut

Reply:
Shall i remove it, since it is in the original commit?

Reply:
Yes, don't let that stop you from improving the code!